android/android.mk

```diff
@@ -21,39 +21,39 @@ output/android_docker.stamp: android/docker/mlperf_mobile/Dockerfile
 	docker image build -t mlcommons/mlperf_mobile_android_builder android/docker/mlperf_mobile
 	touch $@
 
-ANDROID_COMMON_DOCKER_FLAGS1= \
+android_common_docker_flags1= \
```

Comment:
The -it flag doesn't work in jenkins. Throws an error for "TTY mode not supported". Can you please remove this flag as well? It's not required anyways.

Reply:
Fixed.
However, while `-it` does next to nothing for android app make targets, we still need this flag for flutter make targets. Without it sending signals (like pressing ctrl+C for SIGINT) doesn't work.
I added a check for an environment variable to disable this flag in case the command is run in Jenkins.